### ISSUE TITLE
Inconsistent documentation about creationTime

### ISSUE THREAD

**Issue by smousa** [firebase/firebase-tools#3093]
Maybe I am looking at an old version of the documentation and I don't know it yet.

The firestore documentation says that the `creationTime` on the auth.UserMetadata object should be a formatted UTC string of the date.

https://firebase.google.com/docs/reference/admin/node/admin.auth.UserMetadata#creationtime

And if you look at the definition for the `lastRefreshTime` it elaborates on a UTC string as "eg 'Sat, 03 Feb 2001 04:05:06 GMT'"

However, when I verify this by retrieving the `creationTime` from a functions trigger running on my emulator, the resulting value is a string of the millisecond timestamp.

Is that expected?  Is that the same behavior I should expect when running in a live environment?  I haven't tried it live yet, but I will update this ticket if I notice an inconsistency.

What I am ultimately asking for is clarification within the documentation itself, because it is confusing.

Thanks!

**Comment by smousa** [comment]
**update**: I have verified that the live version of firebase auth presents the `creationTime` as `2021-01-29T02:02:11Z`
So it looks like you have divergent behaviors between the emulator and the live system.

**Comment by yuchenshi** [comment]
Would you mind posting some code about how you retrieved the user and `creationTime`? We'd be glad to take a deeper look.

**Comment by smousa** [comment]
Run this cloud function in the emulator and against a real system and you will see the difference 
```javascript
import * as functions from 'firebase-functions';

export default functions.auth.user().onDelete(async (user) => {
  functions.logger.info(user.metadata.creationTime, typeof user.metadata.creationTime)
})
```

**Comment by mdesoto** [comment]
Can confirm that this is still an issue.

**Comment by yuchenshi** [comment]
Hi all, thanks for the input. It is now clear that the problem is with the Firebase Auth emulator and I'll track this as a bug. PRs welcome too. (Previously, I got a bit confused and did not triage this in time since OP mentions "firestore" but this has nothing to do with Firestore. My apologies.)

**Comment by dsl101** [comment]
I posted a [bug report](https://github.com/firebase/firebase-functions/issues/1269) to firebase-functions, as it showed up on upgrading, but I think it might be related to this one. I have an `authUserOnCreate()` trigger, that uses `creationTime`, and this is being passed in as `undefined` since firebase-tools 11.2.0 (I just ran the tests from 10.9.2 for each released tag, and 11.2.0 is where the error first shows up. Might it be related to [this commit](https://github.com/firebase/firebase-tools/commit/44c2eae2fd10d6e635b8ed81f8a4aa2d903e734c)? See line 80 of `src/emulator/auth/state.ts` that was removed.
